pom.xml

```diff
@@ -23,24 +23,25 @@
 
   <properties>
     <java.version>17</java.version>
-    <folio-spring-base.version>7.2.0</folio-spring-base.version>
+    <folio-spring-base.version>8.1.0</folio-spring-base.version>
     <json.version>20210307</json.version>
```

Comment:
<json.version> is not used, propose to delete

Reply:
removed, thanks